crates/sui-indexer-alt-schema/migrations/2024-12-02-185822_add_coin_balance_owner_kind/down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE sum_coin_balances
+DROP COLUMN owner_kind;
+
+ALTER TABLE wal_coin_balances
+DROP COLUMN owner_kind;
```

Comment:
are these new migrations, from experiment findings?

Reply:
These were actually reverted in `main` after I created this PR, so I've removed them here too -- well spotted!